qemu/tests/migration_multi_host.py

```diff
@@ -111,9 +111,9 @@ def check_worker(mig_data):
             if params.get("check_vm_before_migration", "yes") == "no":
                 params["check_vm_needs_restart"] = "no"
 
-            if params.get("start_migration_timeout") == "random":
-                min_t = int(params.get("min_random_timeout"))
-                max_t = int(params.get("max_random_timeout"))
+            if params.get("enable_random_timeout") == "yes":
```

Comment:
yes, but shuang think  params.get("start_migration_timeout") == "random" is not good idea, cause start_migration_timeout should be a int value, so changed.